server/src/main/java/android/system/Os.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (C) 2011 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package android.system;
```

Comment:
Makes sense, let's do an `OsCompat`, as long as it's just one method and not the whole class it should be manageable. And it'll play nicely with the `@*Api` annotations. I'll either do the OO thing again or what they do in AndroidX Core. There has to be an indirection between the call to new API to avoid slow class verification.

Reply:
I will have to put `libcore.io` package in a separate `compileOnly` module and make it work without gradle too. We don't want to ship that inside the package.